### PR TITLE
Please review

### PR DESCRIPTION
**Pull request by Shazmaan**
Missing runtime but will add that soon.

### CODE REVIEW

File: Problem2.cpp [hunk 1]
```diff
@@ -15,32 +15,54 @@ using namespace std;
...
+//                    sum+=nums[index];
+//                }else if(index-2>=0 && find(allNum.begin(),allNum.end(),nums[index-2])!=allNum.end()){
+//                        sum+=nums[index];
+//                }
+////                }
+//            }
+//            cout<<nums[index]<<" "<<sum<<endl;
+//            index+=2;
+//        }
+//        return sum;
```
Comment (super30admin) [inline Problem2.cpp]: This seems be a lot better version as compared to the first solution.

File: Problem2.cpp [hunk 2]
```diff
@@ -15,32 +15,54 @@ using namespace std;
...
+        // Fill remaining positions
+        for (int i=2; i<size; i++)
+        {
+            maxVal = max(nums.at(i)+ind1, ind2);
+            ind1 = ind2;
+            ind2 = maxVal;
         }
-        return sum;
+
+        return maxVal;
```
Comment (super30admin) [inline Problem2.cpp]: This seems be a lot better version as compared to the first solution.

**Comment by jaspindersingh83** [comment]
The DP solution of problem 1 is missing still. Very important. Please submit

**Comment by jaspindersingh83** [comment]
Good

**Pull request closed without merging**